ui/src/chat/ChatEmbedContent/TwitterEmbed.tsx

```diff
@@ -19,7 +19,8 @@ export default function TwitterEmbed({
   const twitterProfilePic = `https://unavatar.io/twitter/${twitterHandle}?fallback=false`;
   const element = document.createElement('html');
   element.innerHTML = embedHtml;
-  const tweetText = element.querySelector('p')?.innerHTML || '';
+  const baseTweetText = element.querySelector('p')?.innerHTML || '';
```

Comment:
Clever, thank you. Now do images 😈

Reply:
(shown if you click the blockquote text)